### PR TITLE
Redirect previous OAuth routes to the new developer lab

### PR DESCRIPTION
**Pull request by connor4312**

### CODE REVIEW

**Comment by connor4312** [comment]
Note that this should, of course, not be deployed until the new frontend

File: src/server/index.js [hunk 1]
```diff
@@ -49,10 +48,7 @@ export default class Server {
     this.app.get("/login/attempt", require('./routes/login').attempt);
     this.app.get("/doc/*", require('./routes/doc'));
 
-    this.app.get("/oauth/manage", require('./routes/react').oauthManage);
-    this.app.post("/oauth/delete/:id", require('./routes/react').oauthDelete);
-    this.app.get("/oauth/edit/:id?", require('./routes/react').oauthEdit);
-    this.app.post("/oauth/edit", require('./routes/react').oauthSave);
+    this.app.all("/oauth*", (req, res) => res.redirect(`${config.url.public}/lab/oauth`));
```
Comment (UnwrittenFun) [inline src/server/index.js]: Shouldn't this use `config.get('url.public')`

Comment (connor4312) [inline src/server/index.js]: Eh, both work :P

Comment (UnwrittenFun) [inline src/server/index.js]: Yeah but for consistency

**Comment by ttaylorr** [comment]
👍

**Comment by ProbablePrime** [comment]
lgtm, we should probably merge and get deployed until the new dev site is ready.

**Comment by SimonSchick** [comment]
Actually, this is related to the oauth management that was moved to FE, this should be deployed about now @ProbablePrime

**Comment by SimonSchick** [comment]
Looks good tho.

**Comment by ProbablePrime** [comment]
What I meant is that I didnt want the new dev site holding this up. We're still erroring on people trying to do oauth from here. Getting this out will send them to the correct place.

**Comment by ProbablePrime** [comment]
Gonna close this, not relevant now.

**Pull request closed without merging**